### PR TITLE
[5.3] Bump Controllers to 29.0.1

### PR DESCRIPTION
**Pull request by gantunesr**
**Description**

Bump controllers module from `28.0.0` to `29.0.1`. The breaking changes are related to the SRP and vault. (`KeyringController`).

[Release Notes 29.0.0](https://github.com/MetaMask/controllers/releases/tag/v29.0.0)
[Release Notes 29.0.1](https://github.com/MetaMask/controllers/releases/tag/v29.0.1)

QA should verify the functionality of,

- [ ] Create a wallet
- [ ] Recover a wallet
- [ ] Lock and unlock a wallet
- [ ] Reveal SRP and Private key

### CODE REVIEW

**Review by rickycodes — APPROVED** [review]
LGTM

File: app/components/Views/ResetPassword/index.js [hunk 1]
```diff
@@ -527,7 +527,9 @@ class ResetPassword extends PureComponent {
     const { KeyringController } = Engine.context;
     const { originalPassword } = this.state;
     const keychainPassword = originalPassword;
-    const mnemonic = await KeyringController.exportSeedPhrase(keychainPassword);
+    const mnemonic = await KeyringController.exportSeedPhrase(
+      keychainPassword,
```
Comment (cortisiko) [inline app/components/Views/ResetPassword/index.js:531]: @gantunesr I noticed there is only one variable defined but I noticed there is a ','. Is that needed? If not carry sir.

Comment (gantunesr) [reply app/components/Views/ResetPassword/index.js:531]: It's a linter rule, you can see this in other parts of the app too. Its not necessary, but the linter will fail if I remove it

Comment (cortisiko) [reply app/components/Views/ResetPassword/index.js:531]: okay got it! Thanks for the explantation

**Comment by plasmacorral** [comment]
### Observation 1

I am seeing asset icons missing for for DAI, LINK, USDC and (not pictured) BAT and MATIC on this branch with Android 12.  I tried to locally build main and these icons are showing. 

Screen Shot 2022-06-01 at 5 10 41 PM

**Comment by plasmacorral** [comment]
### Observation 2
Androi 12, pixel 3a

Invalid password error after deleting wallet and creating a new one, and creating your wallet  just hangs

https://recordit.co/kfLtewtyeT

Reproduction steps for observation 2:
1. Have German language selected and biometrics available but not enabled in MetaMask
2. Have android OS font size set to maximum
3. Create a new wallet
4.  Ignore all prompts to backup SRP
5. Tap burger menu and lock wallet
6. From login page, tap to delete wallet
7. Tap CTA, type "delete" and tap CTA 
8. Once returned to wallet setup tap create new wallet
9. put in new password 2x
10. leave biometric toggle at the default enabled
11. tap checkbox
12. tap CTA
13. note console warning "0 Error; invalid password"
14. app hangs on creating wallet until closed
15. on reopen, new wallet is accessible

**Comment by gantunesr** [comment]
## Observation 1

I see the same behaviour in the main branch



## Observation 2

I'm currently not able to reproduce it

**Comment by plasmacorral** [comment]
I have also not been to reproduce Observation 2, we can monitor for this in regression/post launch

**Comment by plasmacorral** [comment]
Console error was due to user action (multiple taps on back button), and PR is feature QA passed.

**Pull request merged**